tools/uploader-function/src/DicomUploaderFunction/BlobUploader.cs

```diff
@@ -0,0 +1,37 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.IO;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Extensions.Logging;
+using Microsoft.Health.Dicom.Client;
+
+namespace DicomUploaderFunction;
+
+public class BlobUploader
+{
+    private readonly IDicomWebClient _dicomWebClient;
+
+    public BlobUploader(IDicomWebClient dicomWebClient)
+    {
+        _dicomWebClient = dicomWebClient;
+    }
+
+    [FunctionName("BlobUploader")]
+    public async Task Run(
+        [BlobTrigger("%sourceblobcontainer%/{name}", Connection = "sourcestorage")]
```

Comment:
Should this value be scoped to some section? Eg. `"%foo:bar:baz%"`?

Reply:
It's an environment variable so we should be able to, but we'd probably need to use `__` instead of `:`. We'd likely set it to somthing like `sourcestorage__conainer`

Reply:
I use this in [our own functions](https://github.com/microsoft/healthcare-shared-components/blob/f5ef8dd8c2fe80484f44754fd6b3c26cd17e9225/src/Microsoft.Health.Operations.Functions/Management/PurgeOrchestrationInstanceHistory.cs#L27) and the syntax appears to be `:` like configs (I guess to normalize it?)